rig/machine_control/machine_controller.py

```diff
@@ -1427,6 +1460,22 @@ def __init__(self, machine_controller, x, y, start_address, end_address):
         # Current offset from start address
         self._offset = 0
 
+    @property
+    def buffer_size(self):
+        """Return the number of bytes in the write buffer."""
+        return self._write_buffer.buffer_size
+
+    def close(self):
+        """Flush and close the file-like."""
+        if not self._closed:
+            self.flush()
+            self._closed = True
+
+    @property
```

Comment:
Aha! @mundya has just pointed out to me that we might be talking about different things: I was talking about the `closed` property. I presume you are talking about the `buffer_size` property? The GitHub line comment stuff is displayed a bit confusingly in summaries...